backend/loader.py

```diff
@@ -478,14 +482,36 @@ def split_state_dict(sd, additional_state_dicts: list = None):
 
     return state_dict, guess
 
+# To be removed once PR merged on huggingface_guess
+chroma_is_in_huggingface_guess = hasattr(huggingface_guess.model_list, "Chroma")
```

Comment:
this work-around make it work while I wait for the huggingface_guess PR to be merged, I'll clean it up after. code was tested with and without the huggingface_guess changes.